randovania/games/dread/json_data/Dairon.txt

```diff
@@ -1202,6 +1172,8 @@ Extra - asset_id: collision_camera_012
 
 > Lava Floor with Gravity Suit; Heals? False
   * Layers: default
+  > Pickup (Missile Tank Speedboost)
+      Gravity Suit and Ballspark
```

Comment:
This logically doesn't make a difference but IMO probably best to set this to "Gravity suit and morph ball and speedbooster" , for readability sake since this doesn't require an actual ballspark.

Also I don't think ballsparking would ever become an actual trick, but if it ever does, then this spot shouldn't have it.